### PR TITLE
examples/suit_update: clarify README.hardware.md

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

I had the mistake of creating `coaproot` inside the application directory, add a few words to prevent that mistake in the future.

### Testing procedure

Follow the instructions.

### CODE REVIEW

**Comment by chrysn** [comment]
There are a few more mentions of coaproot in examples/suit_update/README.native.md -- should they go as well?

makefiles/suit.inc.mk mentions a SUIT_COAP_FSROOT of that value, what about that?

**Comment by benpicco** [comment]
The instructions for `native` make sense because they also place something in the `coaproot/` directory (and don't attempt a firmware update).

**Review by fjmolinas — CHANGES_REQUESTED** [review]
It is used in `suit.inc.mk`, if you run the auto-test its there.

File: examples/suit_update/README.hardware.md [hunk 1]
```diff
@@ -277,8 +277,7 @@ Devices retrieve the new firmware using the CoAP protocol.
 
 Start `aiocoap-fileserver`:
 
-    $ mkdir -p coaproot
-    $ aiocoap-fileserver coaproot
```
Comment (benpicco) [inline examples/suit_update/README.hardware.md:281]: This was actually correct, I just executed it inside of `examples/suit_update` :sweat_smile:

**Review by fjmolinas — APPROVED** [review]
ACK

**Pull request merged**